cmd/kustomizepatch/runner.go

```diff
@@ -127,13 +133,28 @@ func (r *runner) run(items []*kyaml.RNode) ([]*kyaml.RNode, error) {
 			}
 		}
 
+		var sopsKeysSource string
+		{
+			sopsKeysSource = os.Getenv(sopsKeysSourceEnvVar)
+
+			if sopsKeysSource == "" {
+				sopsKeysSource = "local"
```

Comment:
nit: `"local"` and `"kubernetes"` could be consts.

Reply:
nice idea, will do that, thanks!